src/handlers/sra_handlers.ts

```diff
@@ -70,6 +70,26 @@ export class SRAHandlers {
         const orderbookResponse = await this._orderBook.getOrderBookAsync(page, perPage, baseToken, quoteToken);
         res.status(HttpStatus.OK).send(orderbookResponse);
     }
+    public async orderbookPricesAsync(req: express.Request, res: express.Response): Promise<void> {
+        const { page, perPage } = paginationUtils.parsePaginationConfig(req);
+        const graphUrl = (req.query.graphUrl as string).toLowerCase();
+        const createdBy = req.query.createdBy === undefined ? "" : (req.query.createdBy as string).toLowerCase();
+        const taker = req.query.taker === undefined ? NULL_ADDRESS : (req.query.taker as string).toLowerCase();
+        const feeRecipient = req.query.feeRecipient === undefined ? NULL_ADDRESS : (req.query.feeRecipient as string).toLowerCase();
+        const takerTokenFee: number = Number((req.query.takerTokenFeeAmount as string)) | 0;
```

Comment:
Will setting `takerTokenFee` to zero (if parameters is not provided) result in offers being returned that have 0 fees attached only? Or will it return all orders irrespective of the fee attached? There should be a way for the latter.

Reply:
I see that in `checkBidsOrAsks` the fee filter is only applied if `takerTokenFee` is non-zero. In that case, it will return everything which is good. So nothing to do here

Reply:
One thing: Could you add a comment for an example value of `takerTokenFee`? e.g. 1 corresponds to 0.001%, 1000 corresponds to 1%. This is important to know

Reply:
> One thing: Could you add a comment for an example value of `takerTokenFee`? e.g. 1 corresponds to 0.001%, 1000 corresponds to 1%. This is important to know

> Okay. I will add a comment